### DISCUSSION TITLE
Conda error

### DISCUSSION THREAD

**Discussion by yohanespm77** [teamalgoritma/community#738] [Q&A] [answered]
Saya tidak bisa menggunakan command conda di anaconda prompt
<img width="868" alt="image" src="https://github.com/teamalgoritma/community/assets/106010284/453c2920-5761-4505-abdf-474a5850c075">

Mohon bantuannya kak, terima kasih 🙏

**Comment by amaliasf** [comment, marked as answer]
Dear @yohanespm77 

Berdasarkan screenshot yang dilampirkan seperti nya Anaconda prompt Bapak gagal untuk aktivasi conda environment. 

Berikut solusi yang bisa dilakukan:

1. Sebaiknya untuk nama folder `A S. Puwandren` bisa direname saja menjadi tanpa spasi. Di beberapa kondisi spasi pada nama folder dapat mengganggu proses aktivasi. Ketika sudah di-rename, bisa dicoba untuk dibuka kembali Anaconda Prompt nya.

2. Kalau langkah 1 masih gagal, maka coba jalankan perintah berikut di Anaconda Prompt Anda:
`C:\Users\YOUR_USERNAME\Anaconda3\Scripts\activate`

3. Kalau masih gagal pada tahap 2, bisa dicoba untuk uninstall Anaconda dari direktori C:\Users\YOUR_USERNAME\anaconda3 kemudian restart PC Anda. Kemudian unduh dan install distribusi Anaconda terbaru, **pastikan ketika installer muncul (Select Installation Type), pilih instal untuk for all users**

_Note: Untuk tau apakah proses aktivasi conda environment sudah berhasil adalah sudah muncul tulisan **(base)** pada anaconda prompt tersebut._

Berikut referensi yang dapat ditinjau untuk solusi 2 dan 3: [Reference](https://stackoverflow.com/questions/59721699/anaconda-is-unable-to-create-process-on-windows10 )

Silahkan bisa dicoba terlebih dahulu, dan diinfokan kembali kalau terkendala.

Semoga menjawab :)

**Reply by yohanespm77** [reply]
Baik, terima kasih kak.
